backend/app/controllers/spree/admin/resource_controller.rb

```diff
@@ -76,7 +76,7 @@ def create
   def update_positions
     ActiveRecord::Base.transaction do
       params[:positions].each do |id, index|
-        model_class.find(id).set_list_position(index)
+        model_class.find_by(id: id).try(:set_list_position, index)
```

Comment:
I'm not a big fan of `try`, because it will swallow any errors even if the receiver of the method is _not_ `nil` but it doesn't respond to the method. In this case, it will fail silently if `find_by` somehow returns an object that doesn't respond to `set_list_position` (e.g. because we remove it by mistake, or because something goes wrong with `model_class` and it returns the wrong class).

The correct syntax here would be the safe navigation operator (`&.`), which fails silently if the receiver is `nil` but returns an error in all other scenarios.

Reply:
Good catch, @aldesantis. I agree!

Reply:
ok. the pull request is updated and uses the &. operator.

i thought the more verbose option would be nicer ;). Was not aware of the additional implications of .try vs &.